src/server/input_service.rs

```diff
@@ -352,6 +382,21 @@ fn run_cursor(sp: MouseCursorService, state: &mut StateCursor) -> ResultType<()>
     Ok(())
 }
 
+fn run_window_focus(sp: EmptyExtraFieldService, state: &mut StateWindowFocus) -> ResultType<()> {
+    let window_id = crate::get_focused_window_id();
+    if let Some(window_id) = window_id {
+        if state.is_changed(window_id) {
```

Comment:
When `Follow remote cursor` is not enabled and `Follow remote window` is enabled, move focused window to another display,  it doesn't follow.